src/managementgroup/azext_managementgroup/generated/_params.py

```diff
@@ -0,0 +1,166 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-statements
+
+from azure.cli.core.commands.parameters import (
+    get_three_state_flag,
+    get_enum_type
+)
+
+
+def load_arguments(self, _):
+
+    with self.argument_context('managementgroup management-group list') as c:
+        c.argument('skiptoken', type=str, help='Page continuation token is only used if a previous operation returned '
+                   'a partial result.  If a previous response contains a nextLink element, the value of the nextLink '
+                   'element will include a token parameter that specifies a starting point to use for subsequent '
+                   'calls.')
```

Comment:
The parameter `skiptoken` is for paging, right? If so, please keep consistent with the common paging parameters of other modules. For details, please refer to the code sample [code link](https://github.com/Azure/azure-cli/blob/d04b359a45c7c92485ac590fcdf48caaf44a88b8/src/azure-cli/azure/cli/command_modules/vm/_params.py#L1151-L1152)